completion/available/sdkman.completion.bash

```diff
@@ -8,24 +8,24 @@ function _sdkman_complete() {
 	COMPREPLY=()
 
 	if [ "$COMP_CWORD" -eq 1 ]; then
```

Comment:
If we would stick with `while IFS='' .... < <(compgen ...)`, the variable `line` should be localized.

```suggestion
	local line
	if [ "$COMP_CWORD" -eq 1 ]; then
```